### ISSUE TITLE
Default clearfix implementation has undesirable side effects

### ISSUE THREAD

**Issue by alanhogan** [Compass/compass#974]
I’m tired of seeing focus styles, etc., get unceremoniously clipped like this (see left and bottom edges):

![clipping](http://f.cl.ly/items/2J3M3P150M3r0B3T0x1U/Screen%20Shot%202012-07-26%20at%207.26.39%20PM.png)

I’m going through some code and replacing `@include clearfix` with `@include pie-clearfix` which seems to have fewer side-effects.

Admittedly it is less concise, but it also doesn’t do, you know, _that_.

If I’m not missing anything, I’d like to suggest aliasing the `clearfix` mixin to `pie-clearfix`, and making the current `clearfix` available as `ppk-clearfix` or `oh-clearfix`.

**Comment by chriseppstein** [comment]
The current clearfix used to be pie-clearfix. I changed it to the overflow-based one.

In most cases, the simpler overflow-based clearfix works fine and in these cases uses far less output -- especially for nested selectors. In the other cases, users will see the clipping, read the docs, and choose a more bloated, but more effective mixin for their use case.

TL;DR I don't think it's better to use a bloated mixin as a default than to never see a bug.
